apps/google-analytics-4/lambda/src/controllers/apiController.ts

```diff
@@ -31,7 +31,10 @@ const ApiController = {
     try {
       const serviceAccountKeyFile = req.serviceAccountKey;
       const { propertyId, slug, startDate, endDate, dimensions, metrics } =
-        req.params as unknown as RunReportParamsType;
+        req.query as unknown as RunReportParamsType;
```

Comment:
query vs param as mentioned in PR description. For now they are all queries, open to thoughts and suggestions!